<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Home</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.10/datatables.js"></script>
<link href="site_libs/dt-core-1.10.19/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.19/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.19/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "î‰™";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "î‰™";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ggpackages</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/cgpu/ggpackages">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-12-18
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>ggpackages/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity itâ€™s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191218code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191218)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191218code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191218)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomcgpuggpackagestree5d2c6da359be4ac1be36cb9e7d8e8c39cf045cb8targetblank5d2c6daa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/cgpu/ggpackages/tree/5d2c6da359be4ac1be36cb9e7d8e8c39cf045cb8" target="_blank">5d2c6da</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomcgpuggpackagestree5d2c6da359be4ac1be36cb9e7d8e8c39cf045cb8targetblank5d2c6daa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
working directory clean
</code></pre>
<p>
Note that any generated files, e.g.Â HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If youâ€™ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/cgpu/ggpackages/blob/5d2c6da359be4ac1be36cb9e7d8e8c39cf045cb8/analysis/index.Rmd" target="_blank">5d2c6da</a>
</td>
<td>
cgpu
</td>
<td>
2019-12-18
</td>
<td>
Swaps Rmd (about-&gt;index) where the mini tutorial is hosted
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/cgpu/ggpackages/238022e063df59534d6cd3c5a1ce5f6b79c07f3a/docs/index.html" target="_blank">238022e</a>
</td>
<td>
cgpu
</td>
<td>
2019-12-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/cgpu/ggpackages/f0bfc4b160c237c205d3f49f508b193ee8e3b4d2/docs/index.html" target="_blank">f0bfc4b</a>
</td>
<td>
cgpu
</td>
<td>
2019-12-18
</td>
<td>
Adds first batch of autogenerated {workflowr} files
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/cgpu/ggpackages/blob/01474f658d319bded8b982a311af917ba92c64df/analysis/index.Rmd" target="_blank">01474f6</a>
</td>
<td>
cgpu
</td>
<td>
2019-12-18
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="using-dt-and-githubinstall-r-packages-to-fetch-and-display-in-an-interactive-table-github-hosted-r-packages-containing-the-term-phylogenetic-in-teh-repo-description" class="section level1">
<h1>Using <code>{DT}</code> and <code>{githubinstall}</code> R packages to fetch and display in an interactive table github hosted R packages containing the term <code>phylogenetic</code> in teh repo description</h1>
</div>
<div id="loading-libraries" class="section level1">
<h1>Loading libraries</h1>
<pre class="r"><code>library(githubinstall)
library(DT)
library(skimr)
library(BiocManager)</code></pre>
<pre><code>Bioconductor version 3.9 (BiocManager 1.30.10), ?BiocManager::install for help</code></pre>
<pre><code>Bioconductor version &#39;3.9&#39; is out-of-date; the current release version &#39;3.10&#39;
  is available with R version &#39;3.6&#39;; see https://bioconductor.org/install</code></pre>
</div>
<div id="searching-for-packages-relevant-to-phylogenetic" class="section level1">
<h1>Searching for packages relevant to <code>phylogenetic</code></h1>
<p>The following brief tutorial will guide you through the functions of the <code>{githubinstall}</code> R package, a package to easily query GitHub and find all relevant R packages to your selected query search term of interest. The function <code>githubinstall::gh_search_packages()</code> returns the list of R packages on GitHub that the titles contains a given #keyword .</p>
<p>For example, if you want to search packages that are relevant to <code>phylogenetic</code>, run the following:</p>
<pre class="r"><code>#  Search Packages by a Keyword, in this example we search for &quot;phylogenetic&quot; 
# install.packages(&quot;githubinstall&quot;) || found here: hoxo-m.github.io/githubinstall/

keyword &lt;- &quot;phylogenetic&quot;

library(githubinstall)

pckges    &lt;- githubinstall::gh_search_packages(keyword)</code></pre>
<pre><code>Warning in fread(download_url, sep = &quot;\t&quot;, header = FALSE, stringsAsFactors =
FALSE, : Found and resolved improper quoting out-of-sample. First healed line
4848: &lt;&lt;Puriney honfleuR &quot;Evening, honfleuR&quot; by Seurat&gt;&gt;. If the fields are not
quoted (e.g. field separator does not appear within any field), try quote=&quot;&quot; to
avoid this warning.</code></pre>
<pre class="r"><code>skimr::skim(pckges)</code></pre>
<table>
<caption>Data summary</caption>
<tbody>
<tr class="odd">
<td align="left">Name</td>
<td align="left">pckges</td>
</tr>
<tr class="even">
<td align="left">Number of rows</td>
<td align="left">71</td>
</tr>
<tr class="odd">
<td align="left">Number of columns</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">_______________________</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">Column type frequency:</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">character</td>
<td align="left">3</td>
</tr>
<tr class="odd">
<td align="left">________________________</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">Group variables</td>
<td align="left">None</td>
</tr>
</tbody>
</table>
<p><strong>Variable type: character</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">skim_variable</th>
<th align="right">n_missing</th>
<th align="right">complete_rate</th>
<th align="right">min</th>
<th align="right">max</th>
<th align="right">empty</th>
<th align="right">n_unique</th>
<th align="right">whitespace</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">username</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">5</td>
<td align="right">19</td>
<td align="right">0</td>
<td align="right">60</td>
<td align="right">0</td>
</tr>
<tr class="even">
<td align="left">package_name</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">3</td>
<td align="right">17</td>
<td align="right">0</td>
<td align="right">70</td>
<td align="right">0</td>
</tr>
<tr class="odd">
<td align="left">title</td>
<td align="right">0</td>
<td align="right">1</td>
<td align="right">26</td>
<td align="right">451</td>
<td align="right">0</td>
<td align="right">69</td>
<td align="right">0</td>
</tr>
</tbody>
</table>
<div id="lets-inspect-what-packages-we-retrieved-related-to-phylogenetic" class="section level2">
<h2>Letâ€™s inspect what packages we retrieved related to <code>{phylogenetic}</code></h2>
<pre class="r"><code>DT::datatable(pckges)</code></pre>
<div id="htmlwidget-cd92dd47aae10c008c57" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-cd92dd47aae10c008c57">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71"],["Ax3man","BrunoVilela","DomBennett","GuangchuangYu","GuangchuangYu","GuangchuangYu","Hackout2","JohnsonHsieh","KlausVigo","KlausVigo","LuciaRG","LuciaRotger","OJWatson","Shicheng-Guo","TGuillerme","USCbiostats","USCbiostats","YulongNiu","YulongNiu","adambakewell","andersgs","andrew-hipp","azizka","bdearlove","bomeara","caseywdunn","cboettig","cboettig","cromanpa94","daijiang","darunabas","davidnipperess","dunleavy005","emvolz","ericgoolsby","fkeck","fmichonneau","fmichonneau","gjuggler","grady","heavywatal","heibl","helixcn","hmorlon","joey711","jsilve24","kingaa","ms609","mwpennell","olli0601","pdrhlik","rachelwarnock","reslp","richfitz","richfitz","roblanf","schantepie","shaunpwilkinson","shaunpwilkinson","snoweye","thibautjombart","thibautjombart","ursus-americanus","vanderleidebastiani","vanderleidebastiani","venelin","villandre","vmikk","vnminin","willpearse","zachcp"],["phylopath","sampler","MoreTreeTools","ggtree","tidytree","treeio","epiphylo","iNextPD","phangorn","phangornMCMC","CollessLike","CollessLike","sims4","lyne","mulTree","aphylo","phylogenetic","PhyloProfile","PhyloProfileSuppl","saplings","harrietr","morton","speciesgeocodeR","treeImbalance","utilitree","hutan","pmc","wrightscape","muPHY","phyr","phyloregion","PDcalc","phylomoments","treedater","Rphylopars","phylosignal","chopper","rncl","ggphylo","kdetrees","cellpoptime","ips","phylotools","PANDA","phyloseq","philr","ouch","TreeSearch","arbutus","PANGEA.HIV.sim","phylomice","fossilsim","correlate","diversitree","forest","communitree","IBMphyloepis","pHMM","topdown","phyclust","adephylo","apex","mallorn","PCPS","SYNCSA","POUMM","DMphyClus","phylonest","phylomap","pez","phylogenetictree"],["Perform phylogenetic path analysis in R.","Functions to generate overdispersed and aggregated sampling designs in the geographic, phylogenetic or any trait/variable space.","ðŸ”¼ðŸŒ³ðŸ’» More tools for manipulating phylogenetic trees in R",":christmas_tree:Visualization and annotation of phylogenetic trees",":shower:A Tidy Tool for Phylogenetic Tree Data Manipulation",":seedling:Base Classes and Functions for Phylogenetic Tree Input and Output","Epidemics and outbreak dynamics with phylogenetic trees","iNterpolation and EXTrapolation for phylogenetic diversity","Phylogenetic analysis in R","A simple phylogenetic MCMC program","Methods to compute distribution and percentile of balance indices of phylogenetic trees","Methods to compute distribution and percentile of balance indices of phylogenetic trees","Epidemic case tree simulator and integrated phylogenetic analysis","Modeling DNA methylation dynamics with approaches from phylogenetics.","Performs MCMCglmm on multiple phylogenetic trees","Statistical inference of genetic functions in phylogenetic trees","Statistical inference of genetic functions in phylogenetic trees","A R package for phylogenetic profiling analysis.","A supplementary R package for web-based phylogenetic profiling analysis","Primitive functions for speeding up elements of a phylogenetic comparative workflow","An `R` package for various phylogenetic and evolutionary biology data manipulations","Tools (in R) for manipulation of phylogenetic, genetic, and other biodiversity data","Preparation of species occurrences and distribution data for the use in phylogenetic analyses. SpeciesgeocodeR is built for data exploration and data analysis and especially suited for biogeographical and ecological questions on large datasets. The package includes the easy creation of summary-tables and -graphs and geographical maps, the calculating of coexistence matrices and species ranges (EOO) as well as mapping diversity in geographic areas.","An R package for detecting asymmetry in time-sampled phylogenetic trees.","Some R utilities for dealing with phylogenetic trees, data, analyses","Analysis of phylogenetic tree topology","Phylogenetic Monte Carlo: Quantifying Uncertainty in Phylogenetic Methods (now on CRAN)","Infer adaptive landscapes from phylogenetic trees","Multiple utilities for phylogenetic analyses","Collect and update functions about phylogenetic analyses","An R Package For Putting Phylogenetic Biogeography On The Map","An r package for Phylogenetic Diversity (PD) calculations in ecology, biogeography and conservation biology","Efficient computations of phylogenetic stochastic mapping moments","Scalable relaxed clock phylogenetic dating","Phylogenetic Comparative Tools for Missing Data and Within-Species Variation","Exploring the Phylogenetic Signal in Continuous Traits","chopper -- an R package to manipulate DNA alignments and data preparation for phylogenetic analyses","An R interface to the Nexus Class Library (C++) to read in quickly large phylogenetic trees and data","Phylogenetic plots for R and ggplot","R package for identifying discordant phylogenetic trees in a sample","ðŸš§ Analyze and visualize phylogenetic tree of cell population","R Interfaces to Phylogenetic Software","Phylogenetic tools for Eco-phylogenetics","Phylogenetic ANalyses of DiversificAtion","phyloseq is a set of classes, wrappers, and tools (in R) to make it easier to import, store, and analyze phylogenetic sequencing data; and to reproducibly share that data and analysis with others. See the phyloseq front page:","Phylogenetic Isometric Log Ratio","Ornstein-Uhlenbeck models for phylogenetic comparative hypotheses","Search for phylogenetic trees with custom optimality criterion","Assessing the adequacy of phylogenetic models of quantitative trait evolution","HIV Phylogenetic Simulation Model for the PANGEA-HIV methods comparison exercise.","mice imputation methods using phylogenetic information","R package for simulating fossil data on phylogenetic trees under mechanistic models of preservation and sampling","R package for character correlation analyses with phylogenetic trees","diversitree: comparative phylogenetic analyses of diversification","New Phylogenetics Data Structures in R","analysis of community assembly using phylogenetics in R","IBM with epistasis processes which evolve along a phylogenetic tree","Profile hidden Markov models for phylogenetic analysis","Top-down reconstruction of phylogenetic trees","Phylogenetic Clustering (Phyloclustering)","exploratory analyses for the phylogenetic comparative method","Phylogenetic Methods for Multiple Gene Data","mallorn is an R package for calculating expected Phylogenetic Diversity and Evolutionary Distinctiveness","R Package - PCPS - Principal Coordinates of Phylogenetic Structure","R Package - SYNCSA - Analysis of functional and phylogenetic patterns in metacommunities","The Phylogenetic Ornstein-Uhlenbeck Mixed Model","A R package for phylogenetic clustering","Equivalent numbers for species, phylogenetic or functional diversity in a nested hierarchy of multiple scales","R package for phylogenetic stochastic mapping, including a new method that bypasses matrix exponentiation","Phylogenetics for the Environmental Sciences","phylogenetictree htmlwidget"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>username<\/th>\n      <th>package_name<\/th>\n      <th>title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<p>Above we can see how many packages were found that contain the search term in their description. The <code>githubinstall::gh_search_packages</code> function returns 3 arguments for each package:</p>
<ul>
<li><code>username</code>:account name of package developer on GitHub</li>
<li><code>package_name</code>:the name of the package that you can use as argument in the <code>install.packages()</code> function</li>
<li><code>title</code>: the description of the package</li>
</ul>
</div>
</div>
<div id="mmm-wouldnt-it-be-nice-if-the-package-name-was-actually-a-link.." class="section level1">
<h1>ðŸ¤” Mmm, wouldnâ€™t it be nice if the package name was actually a <clickable> link..</h1>
<p>Letâ€™s try to add some interactivity to our retrieved table and convert the package names to clickable links that will take us to the github repo of the packages.</p>
<div id="reconstructing-the-r-package-github-repo-url-for-clickable-links" class="section level2">
<h2>Reconstructing the R package GitHub repo url for clickable links</h2>
<p>We will recontruct the string with the above pattern for all found packages, and add the html tag for redirecting so the pattern to recreate should look like this:</p>
<pre class="r"><code># Mutate the package names to create clickable links:

pckges$package_name &lt;- paste0(&quot;&lt;a href=&#39;https://github.com/&quot;,
                             pckges$username, 
                             &quot;/&quot;,
                             pckges$package_name, 
                             &quot;&#39;/&quot;,
                             &quot;target=&#39;blank&quot;,
                             &quot;&#39;&gt;&quot;,
                             pckges$package_name, 
                             &quot;&lt;/a&gt;&quot;)</code></pre>
</div>
<div id="updated-datatable-with-clickable-links" class="section level2">
<h2>Updated datatable with clickable links!</h2>
<p>Now we can inspect the packages that seem interesting by clicking the link. This will take us to the respective repository in GitHub.</p>
<pre class="r"><code>DT::datatable(pckges[,-1],      # the -1 to omit the username in the output table
              escape = FALSE   # this is required for the url
             )</code></pre>
<div id="htmlwidget-00724b2efa387e572d7f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-00724b2efa387e572d7f">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71"],["<a href='https://github.com/Ax3man/phylopath'/target='blank'>phylopath<\/a>","<a href='https://github.com/BrunoVilela/sampler'/target='blank'>sampler<\/a>","<a href='https://github.com/DomBennett/MoreTreeTools'/target='blank'>MoreTreeTools<\/a>","<a href='https://github.com/GuangchuangYu/ggtree'/target='blank'>ggtree<\/a>","<a href='https://github.com/GuangchuangYu/tidytree'/target='blank'>tidytree<\/a>","<a href='https://github.com/GuangchuangYu/treeio'/target='blank'>treeio<\/a>","<a href='https://github.com/Hackout2/epiphylo'/target='blank'>epiphylo<\/a>","<a href='https://github.com/JohnsonHsieh/iNextPD'/target='blank'>iNextPD<\/a>","<a href='https://github.com/KlausVigo/phangorn'/target='blank'>phangorn<\/a>","<a href='https://github.com/KlausVigo/phangornMCMC'/target='blank'>phangornMCMC<\/a>","<a href='https://github.com/LuciaRG/CollessLike'/target='blank'>CollessLike<\/a>","<a href='https://github.com/LuciaRotger/CollessLike'/target='blank'>CollessLike<\/a>","<a href='https://github.com/OJWatson/sims4'/target='blank'>sims4<\/a>","<a href='https://github.com/Shicheng-Guo/lyne'/target='blank'>lyne<\/a>","<a href='https://github.com/TGuillerme/mulTree'/target='blank'>mulTree<\/a>","<a href='https://github.com/USCbiostats/aphylo'/target='blank'>aphylo<\/a>","<a href='https://github.com/USCbiostats/phylogenetic'/target='blank'>phylogenetic<\/a>","<a href='https://github.com/YulongNiu/PhyloProfile'/target='blank'>PhyloProfile<\/a>","<a href='https://github.com/YulongNiu/PhyloProfileSuppl'/target='blank'>PhyloProfileSuppl<\/a>","<a href='https://github.com/adambakewell/saplings'/target='blank'>saplings<\/a>","<a href='https://github.com/andersgs/harrietr'/target='blank'>harrietr<\/a>","<a href='https://github.com/andrew-hipp/morton'/target='blank'>morton<\/a>","<a href='https://github.com/azizka/speciesgeocodeR'/target='blank'>speciesgeocodeR<\/a>","<a href='https://github.com/bdearlove/treeImbalance'/target='blank'>treeImbalance<\/a>","<a href='https://github.com/bomeara/utilitree'/target='blank'>utilitree<\/a>","<a href='https://github.com/caseywdunn/hutan'/target='blank'>hutan<\/a>","<a href='https://github.com/cboettig/pmc'/target='blank'>pmc<\/a>","<a href='https://github.com/cboettig/wrightscape'/target='blank'>wrightscape<\/a>","<a href='https://github.com/cromanpa94/muPHY'/target='blank'>muPHY<\/a>","<a href='https://github.com/daijiang/phyr'/target='blank'>phyr<\/a>","<a href='https://github.com/darunabas/phyloregion'/target='blank'>phyloregion<\/a>","<a href='https://github.com/davidnipperess/PDcalc'/target='blank'>PDcalc<\/a>","<a href='https://github.com/dunleavy005/phylomoments'/target='blank'>phylomoments<\/a>","<a href='https://github.com/emvolz/treedater'/target='blank'>treedater<\/a>","<a href='https://github.com/ericgoolsby/Rphylopars'/target='blank'>Rphylopars<\/a>","<a href='https://github.com/fkeck/phylosignal'/target='blank'>phylosignal<\/a>","<a href='https://github.com/fmichonneau/chopper'/target='blank'>chopper<\/a>","<a href='https://github.com/fmichonneau/rncl'/target='blank'>rncl<\/a>","<a href='https://github.com/gjuggler/ggphylo'/target='blank'>ggphylo<\/a>","<a href='https://github.com/grady/kdetrees'/target='blank'>kdetrees<\/a>","<a href='https://github.com/heavywatal/cellpoptime'/target='blank'>cellpoptime<\/a>","<a href='https://github.com/heibl/ips'/target='blank'>ips<\/a>","<a href='https://github.com/helixcn/phylotools'/target='blank'>phylotools<\/a>","<a href='https://github.com/hmorlon/PANDA'/target='blank'>PANDA<\/a>","<a href='https://github.com/joey711/phyloseq'/target='blank'>phyloseq<\/a>","<a href='https://github.com/jsilve24/philr'/target='blank'>philr<\/a>","<a href='https://github.com/kingaa/ouch'/target='blank'>ouch<\/a>","<a href='https://github.com/ms609/TreeSearch'/target='blank'>TreeSearch<\/a>","<a href='https://github.com/mwpennell/arbutus'/target='blank'>arbutus<\/a>","<a href='https://github.com/olli0601/PANGEA.HIV.sim'/target='blank'>PANGEA.HIV.sim<\/a>","<a href='https://github.com/pdrhlik/phylomice'/target='blank'>phylomice<\/a>","<a href='https://github.com/rachelwarnock/fossilsim'/target='blank'>fossilsim<\/a>","<a href='https://github.com/reslp/correlate'/target='blank'>correlate<\/a>","<a href='https://github.com/richfitz/diversitree'/target='blank'>diversitree<\/a>","<a href='https://github.com/richfitz/forest'/target='blank'>forest<\/a>","<a href='https://github.com/roblanf/communitree'/target='blank'>communitree<\/a>","<a href='https://github.com/schantepie/IBMphyloepis'/target='blank'>IBMphyloepis<\/a>","<a href='https://github.com/shaunpwilkinson/pHMM'/target='blank'>pHMM<\/a>","<a href='https://github.com/shaunpwilkinson/topdown'/target='blank'>topdown<\/a>","<a href='https://github.com/snoweye/phyclust'/target='blank'>phyclust<\/a>","<a href='https://github.com/thibautjombart/adephylo'/target='blank'>adephylo<\/a>","<a href='https://github.com/thibautjombart/apex'/target='blank'>apex<\/a>","<a href='https://github.com/ursus-americanus/mallorn'/target='blank'>mallorn<\/a>","<a href='https://github.com/vanderleidebastiani/PCPS'/target='blank'>PCPS<\/a>","<a href='https://github.com/vanderleidebastiani/SYNCSA'/target='blank'>SYNCSA<\/a>","<a href='https://github.com/venelin/POUMM'/target='blank'>POUMM<\/a>","<a href='https://github.com/villandre/DMphyClus'/target='blank'>DMphyClus<\/a>","<a href='https://github.com/vmikk/phylonest'/target='blank'>phylonest<\/a>","<a href='https://github.com/vnminin/phylomap'/target='blank'>phylomap<\/a>","<a href='https://github.com/willpearse/pez'/target='blank'>pez<\/a>","<a href='https://github.com/zachcp/phylogenetictree'/target='blank'>phylogenetictree<\/a>"],["Perform phylogenetic path analysis in R.","Functions to generate overdispersed and aggregated sampling designs in the geographic, phylogenetic or any trait/variable space.","ðŸ”¼ðŸŒ³ðŸ’» More tools for manipulating phylogenetic trees in R",":christmas_tree:Visualization and annotation of phylogenetic trees",":shower:A Tidy Tool for Phylogenetic Tree Data Manipulation",":seedling:Base Classes and Functions for Phylogenetic Tree Input and Output","Epidemics and outbreak dynamics with phylogenetic trees","iNterpolation and EXTrapolation for phylogenetic diversity","Phylogenetic analysis in R","A simple phylogenetic MCMC program","Methods to compute distribution and percentile of balance indices of phylogenetic trees","Methods to compute distribution and percentile of balance indices of phylogenetic trees","Epidemic case tree simulator and integrated phylogenetic analysis","Modeling DNA methylation dynamics with approaches from phylogenetics.","Performs MCMCglmm on multiple phylogenetic trees","Statistical inference of genetic functions in phylogenetic trees","Statistical inference of genetic functions in phylogenetic trees","A R package for phylogenetic profiling analysis.","A supplementary R package for web-based phylogenetic profiling analysis","Primitive functions for speeding up elements of a phylogenetic comparative workflow","An `R` package for various phylogenetic and evolutionary biology data manipulations","Tools (in R) for manipulation of phylogenetic, genetic, and other biodiversity data","Preparation of species occurrences and distribution data for the use in phylogenetic analyses. SpeciesgeocodeR is built for data exploration and data analysis and especially suited for biogeographical and ecological questions on large datasets. The package includes the easy creation of summary-tables and -graphs and geographical maps, the calculating of coexistence matrices and species ranges (EOO) as well as mapping diversity in geographic areas.","An R package for detecting asymmetry in time-sampled phylogenetic trees.","Some R utilities for dealing with phylogenetic trees, data, analyses","Analysis of phylogenetic tree topology","Phylogenetic Monte Carlo: Quantifying Uncertainty in Phylogenetic Methods (now on CRAN)","Infer adaptive landscapes from phylogenetic trees","Multiple utilities for phylogenetic analyses","Collect and update functions about phylogenetic analyses","An R Package For Putting Phylogenetic Biogeography On The Map","An r package for Phylogenetic Diversity (PD) calculations in ecology, biogeography and conservation biology","Efficient computations of phylogenetic stochastic mapping moments","Scalable relaxed clock phylogenetic dating","Phylogenetic Comparative Tools for Missing Data and Within-Species Variation","Exploring the Phylogenetic Signal in Continuous Traits","chopper -- an R package to manipulate DNA alignments and data preparation for phylogenetic analyses","An R interface to the Nexus Class Library (C++) to read in quickly large phylogenetic trees and data","Phylogenetic plots for R and ggplot","R package for identifying discordant phylogenetic trees in a sample","ðŸš§ Analyze and visualize phylogenetic tree of cell population","R Interfaces to Phylogenetic Software","Phylogenetic tools for Eco-phylogenetics","Phylogenetic ANalyses of DiversificAtion","phyloseq is a set of classes, wrappers, and tools (in R) to make it easier to import, store, and analyze phylogenetic sequencing data; and to reproducibly share that data and analysis with others. See the phyloseq front page:","Phylogenetic Isometric Log Ratio","Ornstein-Uhlenbeck models for phylogenetic comparative hypotheses","Search for phylogenetic trees with custom optimality criterion","Assessing the adequacy of phylogenetic models of quantitative trait evolution","HIV Phylogenetic Simulation Model for the PANGEA-HIV methods comparison exercise.","mice imputation methods using phylogenetic information","R package for simulating fossil data on phylogenetic trees under mechanistic models of preservation and sampling","R package for character correlation analyses with phylogenetic trees","diversitree: comparative phylogenetic analyses of diversification","New Phylogenetics Data Structures in R","analysis of community assembly using phylogenetics in R","IBM with epistasis processes which evolve along a phylogenetic tree","Profile hidden Markov models for phylogenetic analysis","Top-down reconstruction of phylogenetic trees","Phylogenetic Clustering (Phyloclustering)","exploratory analyses for the phylogenetic comparative method","Phylogenetic Methods for Multiple Gene Data","mallorn is an R package for calculating expected Phylogenetic Diversity and Evolutionary Distinctiveness","R Package - PCPS - Principal Coordinates of Phylogenetic Structure","R Package - SYNCSA - Analysis of functional and phylogenetic patterns in metacommunities","The Phylogenetic Ornstein-Uhlenbeck Mixed Model","A R package for phylogenetic clustering","Equivalent numbers for species, phylogenetic or functional diversity in a nested hierarchy of multiple scales","R package for phylogenetic stochastic mapping, including a new method that bypasses matrix exponentiation","Phylogenetics for the Environmental Sciences","phylogenetictree htmlwidget"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>package_name<\/th>\n      <th>title<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]}},"evals":[],"jsHooks":[]}</script>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.0 (2019-04-26)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.5

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] BiocManager_1.30.10 skimr_2.0.2         DT_0.10            
[4] githubinstall_0.2.2 workflowr_1.5.0    

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3        tidyr_1.0.0       prettyunits_1.0.2 ps_1.3.0         
 [5] assertthat_0.2.1  zeallot_0.1.0     rprojroot_1.3-2   digest_0.6.23    
 [9] mime_0.7          R6_2.4.1          repr_1.0.2        backports_1.1.5  
[13] evaluate_0.14     httr_1.4.1        highr_0.8         pillar_1.4.2     
[17] rlang_0.4.2       curl_4.3          data.table_1.12.8 whisker_0.4      
[21] callr_3.4.0       rmarkdown_2.0     desc_1.2.0        devtools_2.2.1   
[25] stringr_1.4.0     htmlwidgets_1.5.1 shiny_1.4.0       compiler_3.6.0   
[29] httpuv_1.5.2      xfun_0.11         pkgconfig_2.0.3   base64enc_0.1-3  
[33] pkgbuild_1.0.6    htmltools_0.4.0   tidyselect_0.2.5  tibble_2.1.3     
[37] fansi_0.4.0       crayon_1.3.4      dplyr_0.8.3       withr_2.1.2      
[41] later_1.0.0       xtable_1.8-4      jsonlite_1.6      lifecycle_0.1.0  
[45] git2r_0.26.1      magrittr_1.5      cli_2.0.0         stringi_1.4.3    
[49] fs_1.3.1          promises_1.1.0    remotes_2.1.0     testthat_2.3.1   
[53] ellipsis_0.3.0    vctrs_0.2.1       tools_3.6.0       glue_1.3.1       
[57] purrr_0.3.3       crosstalk_1.0.0   fastmap_1.0.1     processx_3.4.1   
[61] pkgload_1.0.2     yaml_2.2.0        sessioninfo_1.1.1 memoise_1.1.0    
[65] knitr_1.26        usethis_1.5.1    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
